src/client/layouts/components/admin/Sidebar.tsx

```diff
@@ -1,55 +1,102 @@
-import React from 'react';
-import styled from 'styled-components';
-import { TESC_BLUE, BORDER_RADIUS_LG } from '~/styles/constants';
-import AdminNav from './AdminNav';
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import { TESC_BLUE, TESC_BLUE_LIGHTER, BORDER_RADIUS_LG } from '~/styles/constants';
 import { Rounded } from '~/styles';
+import { breakpoints } from '~/styles/breakpoints';
 import { NavLink } from 'react-router-dom';
 
 const Container = styled(Rounded)`
-  min-width: 17rem;
-  max-width: 17rem;
   background: ${TESC_BLUE};
   color: white;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 1.5rem;
+
+  @media (min-width: ${breakpoints['md']}) {
+    min-width: 17rem;
+    max-width: 17rem;
+  }
 `;
 
-const activeClassName = 'nav-item-active'
+const LogoContainer = styled.div.attrs(props => ({
+  className: `flex-column text-white ${props.className}`
+}))`
+  img {
+    max-height: 5rem;
+    width: auto;
+  }
+`
 
-const Link = styled(NavLink).attrs({
-  activeClassName
-})`
+const ListElements = styled.div<{
+  isOpen: boolean;
+}>`
+  ${props => !props.isOpen && css`
+    display: none;
```

Comment:
this seems to be triggering the items to not show. It makes sense - isOpen = false is the default init state

Reply:
is true meant to be the init state?

Reply:
Oh ummm I meant to add a media query to make sure it only hid them for mobile. Let me fix